### DISCUSSION TITLE
实时程序写入doris数据丢失问题

### DISCUSSION THREAD

**Discussion by Fang-gg** [apache/doris#25520] [A - General / Q&A]
实时程序，sparkstreaming以DataFrame方式写入doris，当一批数据写入时，此时doris集群宕机进行重启之后，该批次数据就会丢失掉，请问下有什么好的方式可以将这部分丢失数据回补进doris中
(Real-time program, sparkstreaming writes doris in DataFrame mode. When a batch of data is written, the batch of data will be lost after doris cluster breaks down and restarts. May I ask if there is any good way to fill this part of lost data back into doris)

**Comment by LemonLiTree** [comment]
Why does doris go down?
